api/src/main/java/org/eclipse/microprofile/openapi/annotations/Components.java

```diff
@@ -36,7 +36,7 @@
  * Describes the Components object that holds various reusable objects for different aspects of the OpenAPI
  * Specification (OAS).
  *
- * @see <a href= "https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.0.md#componentsObject"> OpenAPI
+ * @see <a href= "https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.1.0.md#componentsObject"> OpenAPI
```

Comment:
It looks like the IDs for the subsections are now camel case. I only checked a couple of them, but I expect it affects them all...
```suggestion
 * @see <a href= "https://github.com/OAI/OpenAPI-Specification/blob/main/versions/3.1.0.md#components-object"> OpenAPI
```